dnf-docker-test/features/steps/file_steps.py

```diff
@@ -0,0 +1,85 @@
+from __future__ import absolute_import
+from __future__ import unicode_literals
+
+from behave import given
+from behave.model import Table
+import six
+from six.moves import configparser
+
+import file_utils
+import table_utils
+
+HEADINGS_INI = ["Section", "Key", "Value"]
+
+def conf2table(configuration):
+    """
+    :param configparser.ConfigParser configuration: Configuration
+    :return: Table
+    :rtype: behave.model.Table
+    """
+    table = Table(HEADINGS_INI)
+    def s2s(s):
+        return six.text_type(s, "utf-8") if six.PY2 else s
+    for section in configuration.sections():
+        for key, value in configuration.items(section):
+            table.add_row([s2s(section), s2s(key), s2s(value)])
+    return table
+
+@given('a file "{filepath}" with')
```

Comment:
no, you provide multiline text, though I forgot to write example in documentation below